README.rst

```diff
@@ -52,6 +52,8 @@ Setup
 
 For backwards compatibility you can also create your own file and pass it using ``--file settings.ini``.
 
+If you want to transfer liked songs from Spotify, choose "yes" for oAuth authentification, otherwise choose "no".
```

Comment:
```suggestion
If you want to transfer private playlists from Spotify (i.e. liked songs), choose "yes" for oAuth authentication, otherwise choose "no".
```